bears/python/PyUnusedCodeBear.py

```diff
@@ -15,7 +15,7 @@ class PyUnusedCodeBear(LocalBear):
     CAN_DETECT = {'Unused Code'}
 
     def run(self, filename, file,
-            remove_all_unused_imports: bool=False,
+            remove_all_unused_imports: bool=True,
             remove_unused_variables: bool=True):
         """
         Detects unused code. By default this functionality is limited to:
```

Comment:
You also have to adapt the doc string to this change. Currently it states that by default only unneeded builtin imports are handled. And `:param remove_all_unused_imports:` should then better explain what happens in case of `False`